components/fieldgroup/stories/fieldgroup.test.js

```diff
@@ -7,47 +6,51 @@ export const FieldGroupSet = Variants({
 	testData: [
 		{
 			testHeading: "Default",
+			inputType: "radio",
 		},
 		{
-			testHeading: "Horizontal",
+			testHeading: "Vertical checkboxes",
+			layout: "vertical",
+			inputType: "checkbox",
+			helpText: "Make a selection.",
```

Comment:
To me, I don't think we need to set the `helpText` either! We could just depend on the `helpText` value from the default args and then remove these "Make a selection"s.